Skype/Trusted-Application-API/SDK/ClientModel/Resources/AudioVideoFlow.cs

```diff
@@ -91,6 +91,28 @@ public FlowState State
             return await tcs.Task.ConfigureAwait(false);
         }
 
+        public async Task StopPromptsAsync(LoggingContext loggingContext)
+        {
+            string href = PlatformResource?.StopPromptsLink?.Href;
+            if (string.IsNullOrWhiteSpace(href))
+            {
+                throw new CapabilityNotAvailableException("Link to stop prompts is not available.");
+            }
+
+            var stopPromptLink = UriHelper.CreateAbsoluteUri(BaseUri, href);
+
+            TaskCompletionSource<Prompt> tcs = new TaskCompletionSource<Prompt>();
+            var response = await PostRelatedPlatformResourceAsync(stopPromptLink, null, loggingContext).ConfigureAwait(false);
```

Comment:
When does this HTTP request return? When all the prompts are signaled to be cancelled or when they have actually been canceled/removed?